src/Exception/MerchantTimeToWait.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
💅  I think it's a good practice to add the word Exception at the end of the Files to improve readability.

`MerchantTimeToWaitException` is clearer than `MerchantTimeToWait`

Reply:
Similar to the previous comment, maybe we can discuss this a bit more.